api/overview/azure/search.documents-readme.md

```diff
@@ -440,8 +460,8 @@ additional questions or comments.
 [azure_sub]: https://azure.microsoft.com/free/
 [RequestFailedException]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/core/Azure.Core/src/RequestFailedException.cs
 [status_codes]: https://docs.microsoft.com/rest/api/searchservice/http-status-codes
-[samples]: https://github.com/azure/azure-sdk-for-net/tree/Azure.Search.Documents_11.0.0/sdk/search/Azure.Search.Documents/samples
-[search_contrib]: https://github.com/azure/azure-sdk-for-net/tree/Azure.Search.Documents_11.0.0/sdk/search/CONTRIBUTING.md
+[samples]: https://github.com/Azure/azure-sdk-for-net/tree/035ff22811a72bdd193a034270247a64f8421c61/sdk/search/Azure.Search.Documents/samples
```

Comment:
Interesting.  I missed this one when we were making all the URIs absolute.  I wonder why it's not using the release tag though?  We'll make it an absolute URI for next release either way.